hlf/chaincode/operation.md

```diff
@@ -75,22 +60,16 @@ The Errors returned are defined [here](errors.md#Errors).
        - Description: This error is returned, if the given parameters could not be parsed. If some attributes are not well formatted, they are listed in invalidParams.  
        For detailed informations see [Input Checks](#Checks).
 
-
-### GetOperationData
-- ID = getOperationData
+### RejectOperation

```

Comment:
should throw an error if the user that wants to reject is not on the missing approval list/groups

Reply:
see #106